src/components/strategies/useUpdateStrategy.ts

```diff
@@ -7,19 +7,24 @@ import {
   useQueryClient,
   useUpdateStrategyQuery,
 } from 'libs/queries';
-import { PathNames, useNavigate } from 'libs/routing';
+import { useNavigate } from 'libs/routing';
 import { useWeb3 } from 'libs/web3';
+import { useState } from 'react';
+import { ONE_AND_A_HALF_SECONDS_IN_MS } from 'utils/time';
+import { handleTxStatusAndRedirectToOverview } from './create/utils';
 
 export const useUpdateStrategy = () => {
   const { user } = useWeb3();
   const { dispatchNotification } = useNotifications();
   const updateMutation = useUpdateStrategyQuery();
   const cache = useQueryClient();
   const navigate = useNavigate<MyLocationGenerics>();
+  const [isProcessing, setIsProcessing] = useState<boolean>(false);
 
   const pauseStrategy = async (
     strategy: Strategy,
-    successEventsCb?: () => void
+    successEventsCb?: () => void,
+    closeModalCb?: () => void
```

Comment:
@pingustar we talked about passing modal id and close the use closeModal from useModal. But I think that my closeModalCb is valid too and maybe better because we already have the modal id when calling pause strategy. let me know what u think